frontend/src/components/TournamentManager.js

```diff
@@ -298,6 +346,38 @@ const TournamentManager = () => {
     return `${hours}:${minutes}`;
   };
 
+  const submitPools = () => {
+    if (!isPoolsEdited()) {
+      return;
+    }
+
+    let createdPools = [];
+    const createdPoolNames = Object.keys(pools).filter(
+      name => name !== "Remaining"
+    );
+    for (const [index, poolName] of createdPoolNames.entries()) {
+      // don't send empty pools
+      if (!pools[poolName]) continue;
+
+      let poolSeeds = [];
+      for (let { seed } of pools[poolName]) {
+        poolSeeds.push(seed);
+      }
+      createdPools.push({
+        name: poolName,
+        seeding: poolSeeds,
+        sequence_number: index + 1
```

Comment:
Here index will be with respect to just newly created pools right? But we need to make sure this index is unique with old pools too

Reply:
no, while creating the pools store in TournamentManager, i'm adding the previously created pools also
```js
// add previously created pools
    for (let { name, initial_seeding } of alreadyCreatedPools) {
        let poolTeams = [];
        Object.keys(initial_seeding).forEach(seed => {
          poolTeams.push({ seed, name: teamsMap()[tournamentSeeding()[seed]] });
        });
        setPools(name, poolTeams);
      }
```

Reply:
this is fine right ?